README.md

```diff
@@ -14,7 +14,9 @@ Read [CONTRIBUTING](CONTRIBUTING.md) for build and test instructions.
 * `macvlan`: Creates a new MAC address, forwards all traffic to that to the container.
 * `ptp`: Creates a veth pair.
 * `vlan`: Allocates a vlan device.
-
+#### Windows: windows specific
+* `win-l2bridge`: Creates a bridge, adds the host and the container to it.
```

Comment:
Let's change this from "win-l2bridge" to just "win-bridge", since the HNS stuff the bridge relies on does things from L2 to L4 and above.